Add Find-PHP-Vulnerabilities

WangYihang opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

Short Desc:
```
A plug-in of sublime 2/3 which is able to find PHP vulnerabilities
```
Test Case: 
![image](https://user-images.githubusercontent.com/16917636/38801937-bbb330e2-419d-11e8-85d5-5e1e49593e15.png)

WangYihang commented:
@packagecontrol-bot Key map override bug fixed, please recheck, thank you

WangYihang commented:
WTF ?????

WangYihang commented:
How to solve this shit error!!! Fuck

Thom1729 commented:
How do you decide what a vulnerability is? Is there a standard for this? From the readme, it is totally unclear what kinds of vulnerabilities this will detect. In addition, the regexp-based approach does not look very reliable, and there are no tests. Are there existing PHP linters that may provide a superset of this functionality?

I see that all of the output is in (I think) Chinese. I'm not sure whether there's an actual rule about language, but as far as I know the channel maintainers don't read Chinese.

Is inserting comments really the way to go here? The simple alternative would be printing to an output panel. A more complicated approach would be to use phantoms.

WangYihang commented:
@FichteFoll Hi, Fichte~ I got a problem, just now I updated the documentation of this plugin, and run `git tag` command, then push the new commit to GitHub...
But I tried to search in sublime plug repo, but can not find this repo...
Should I open a new pull request everytime I updated my plugins?

FichteFoll commented:
It may take up to 1 hour for your new tag to get picked up by the crawler.

WangYihang commented:
got it, the problem is solved after waiting for a  few minutes, thank you
bro

FichteFoll <notifications@github.com>于2018年6月14日周四 上午8:39写道：

> It may take up to 1 hour for your new tag to get picked up by the crawler.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/wbond/package_control_channel/pull/7060#issuecomment-397130797>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQIkhJ5TlVWQ8JPFsxmLNfvO_Hb7871tks5t8bC6gaJpZM4TWMRw>
> .
>
